docs/src/pages/careers/react-engineer-core.md

```diff
@@ -0,0 +1,126 @@
+
+## Details of the Role
+
+- Location: Remote (preference for UTC-6 to UTC+5).
+- Type of work: Full-time (contractor or employee [depending on circumstances](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501))
+- Start date: Immediately.
+- Level: [4 or above](https://docs.google.com/spreadsheets/d/1dDdPD-flNXlgZ0E3ZxVvCDx27RFuhVWJrcfcjNu_I8k/edit#gid=0).
+- We're a remote company, we prefer asynchronous communication over meetings.
+- We work independently, the rest of us won't know what you're doing day-to-day unless you tell us.
+
+## About the company
+
+MUI started with Material UI, the most successful React implementation of Google's Material Design.
+It has gained a large following, not only due to the fidelity to Material Design, but also because of the number of components, its carefully designed component API, obsession for details, and community engagement.
+Today, countless teams and organizations rely on our open-source libraries to build their design system.
+
+A couple of years ago, we started to expand our suite of products.
+We released [MUI X](/x/), a collection of advanced components; [MUI Design kits](/design-kits/), the MUI components available for the most popular design tools; and also host [Templates](/templates/), a set of pre-built UI kits.
+We've seen rapid growth with all of them (2-3x per year) and we have more to come.
+
+We are a fully distributed team, spread across multiple timezones.
+We work mainly asynchronously, relying mostly on written collaboration.
+Every contributor has the freedom to define how they work — the rest of us won't know what you're doing day-to-day unless you tell us.
+For additional details about the MUI team and culture, you can check our [careers](/careers/) and [about](/about/) pages and also our [public Handbook](https://mui-org.notion.site/Handbook-f086d47e10794d5e839aef9dc67f324b).
+
+## Why we're hiring
+
+Both our open-source community and our premium products are growing fast (x2-3 YoY).
+We need talented people to keep that going!
+
+The core components team (MUI Core) needs help.
+They are working on 4 products (Material UI, MUI Base, Joy UI, and MUI System) with a tiny team.
+
+We also need help to continue to improve the health of the open-source product: make the core components easier to use, engage more with the community, polish all the details, make the components more accessible, guiding developers to answers, and just generally being a positive presence in the open-source community.
+
+## About the role
+
+### Why this is interesting
+
+Our solution empowers React developers to build awesome applications faster – we see over a million developers on MUI's docs every month. – hundreds of thousands of developers use MUI every month.
+
+The core components are massively used, you will receive a lot of feedback from your work. The community has high expectations, they will push you to do better, every day.
```

Comment:
```suggestion
The core components are widely used, you will receive a lot of feedback from your work. The community has high expectations, they will push you to do better, every day.
```

Reply:
```suggestion
The core components are widely used, you will receive a lot of feedback for your work. The community has high expectations, they will push you to do better, every day.
```

Should we also replace "from" with "for"?

Reply:
"on", I would argue. e.g.

https://eur-lex.europa.eu/LexUriServ/LexUriServ.do?uri=COM:2004:0158:FIN:EN:PDF

<img width="645" alt="Screenshot 2022-05-25 at 02 06 26" src="https://user-images.githubusercontent.com/3165635/170151452-c8734006-088f-41a4-b574-91fa6b297c2f.png">